include/real/real_data.hpp

```diff
@@ -14,6 +14,7 @@
 #include <real/real_exception.hpp>
 #include <real/real_rational.hpp>
 #include <real/integer_number.hpp>
+#include <real/real_math.hpp>
 
 namespace boost { 
```

Comment:
We should keep all interfaces for this functions equal to those in the STL for other datatypes so they can be drop in replacements

Reply:
So, should we replace the file name to "math.hpp"(STL interface), so the user can import it like "real/math.hpp"? or you were talking about the interface of functions stored in this file.